### DISCUSSION TITLE
Syntax error when loading preview of components in react

### DISCUSSION THREAD

**Discussion by simbamufasa** [fwouts/previewjs#1215] [Q&A]
Hi.

I'm new to preview.js and I'm trying to configure it for my react project. However, even after setting up the wrapper as I am using some providers, the same error is still persisting.

`SyntaxError: The requested module '/preview/src/config/default.js' does not provide an export named 'endpoint'`

Preview is throwing that error whenever I try and load any component from my project. Any help would be appreciated as I can't find anything on this online.

**Comment by fwouts** [comment]
Hey @simbamufasa, would you have a sample repo to reproduce? It sounds like there's an import of "endpoint" somewhere that can't be handled by Preview.js for some reason, but it's hard to know more without seeing the content of `src/config/default.js` for example.
